datafusion/core/tests/sql/aggregates.rs

```diff
@@ -1815,11 +1815,11 @@ async fn aggregate_avg_add() -> Result<()> {
     assert_eq!(results.len(), 1);
 
     let expected = vec![
-        "+--------------+----------------------------+----------------------------+----------------------------+",
-        "| AVG(test.c1) | AVG(test.c1) Plus Int64(1) | AVG(test.c1) Plus Int64(2) | Int64(1) Plus AVG(test.c1) |",
-        "+--------------+----------------------------+----------------------------+----------------------------+",
-        "| 1.5          | 2.5                        | 3.5                        | 2.5                        |",
-        "+--------------+----------------------------+----------------------------+----------------------------+",
+        "+--------------+---------------------------------------------+---------------------------------------------+---------------------------------------------+",
+        "| AVG(test.c1) | AVG(test.c1) Plus CAST(Int64(1) AS Float64) | AVG(test.c1) Plus CAST(Int64(2) AS Float64) | CAST(Int64(1) AS Float64) Plus AVG(test.c1) |",
```

Comment:
The changes of header name makes me confusion and weird, although i know why it changed.
I think the original display is more meaningful.

And I also check the display of the header:
```
spark-sql> desc t1;
22/08/30 18:16:27 WARN ObjectStore: Failed to get database global_temp, returning NoSuchObjectException
col_name	data_type	comment
c1                  	int
Time taken: 0.111 seconds, Fetched 1 row(s)
spark-sql> select c1=cast(1 as float) from t1;
(c1 = CAST(1 AS FLOAT))
false
Time taken: 0.093 seconds, Fetched 1 row(s)
```

Do we have a good way and don't change display of the header?

Reply:
I'll need to look into this some more. I was also confused why this changed.

Reply:
I looked at Spark and Postgres.

Spark:

```
scala> spark.sql("select id*2, cast(id as string), name from t").show()
+--------+---+-----+
|(id * 2)| id| name|
+--------+---+-----+
|       2|  1|hello|
|       4|  2|    x|
+--------+---+-----+
```

Postgres:

```
postgres=# select id*2, cast(id as varchar(12)), name from t1;
 ?column? | id | name 
----------+----+------
        2 | 1  | x
```

In the case of `CAST`, neither changes the name of the resulting column. This makes sense to me and I will make that change here as well.

Reply:
This is now fixed